projects/packages/search/src/dashboard/components/record-meter/notice-box.jsx

```diff
@@ -19,10 +25,35 @@ const CLOSE_TO_LIMIT_PERCENT = 0.8; //TODO: currently 'close' is defined as 80%.
  */
 export function NoticeBox( props ) {
 	const notices = [];
+	const [ showNotice, setShowNotice ] = useState( true );
+
+	// deal with localStorage for ensuring dismissed notice boxs are not re-displayed
+	const dismissedNoticesString = localStorage.getItem( 'dismissedNoticeBoxes' )
+		? localStorage.getItem( 'dismissedNoticeBoxes' )
+		: '';
+
+	const dismissedNoticesArray =
+		dismissedNoticesString.length > 0 ? dismissedNoticesString.split( ',' ) : [];
+
+	const dismissNoticeBox = () => {
+		setShowNotice( false );
+
+		if ( dismissedNoticesString ) {
+			if ( dismissedNoticesArray.includes( notices[ 0 ].id ) ) {
+				localStorage.setItem(
+					'dismissedNoticeBoxes',
+					dismissedNoticesString + ',' + notices[ 0 ].id
+				);
+			}
+		} else {
+			localStorage.setItem( 'dismissedNoticeBoxes', notices[ 0 ].id );
+		}
+	};
 
 	// check data is valid
 	if ( props.hasValidData === false ) {
 		notices.push( {
+			id: 1,
```

Comment:
Using ID is a good idea, I'd go for this as well 👍 
To make it even better, could we maybe use some constants to make the magic number more meaningful. 

```
const CONTENT_NOT_FOUND = 1;
```

Reply:
nice one! It is even better, so much more readable now 😄